MasterServer/Program.cs

```diff
@@ -6,114 +6,146 @@
 
 namespace MasterServer
 {
-	class Program
+	internal static class Program
 	{
+		private static bool s_Daemon = false;
+		private static string s_Host = "0.0.0.0";
+		private static ushort s_Port = 15940;
+		private static int s_EloRange = 0;
+
+		private static MasterServer s_Server;
+
 		private static void Main(string[] args)
 		{
-			string host = "0.0.0.0";
-			ushort port = 15940;
-			string read = string.Empty;
-			int eloRange = 0;
+			ParseArgs(args);
+
+			Console.WriteLine("Hosting ip [{0}] on port [{1}]", s_Host, s_Port);
+			PrintHelp();
+
+			s_Server = new MasterServer(s_Host, s_Port)
+			{
+				EloRange = s_EloRange
+			};
+			s_Server.ToggleLogging();
+
+			while (true)
+			{
+				if (!s_Daemon)
+					HandleConsoleInput();
+			}
+		}
 
+		private static void ParseArgs(string[] args)
```

Comment:
Same as before. Just to make it more readable I'd rename this to `ParseArguments`

Reply:
Done